src/bootstrap/Cargo.toml

```diff
@@ -14,16 +14,6 @@ name = "bootstrap"
 path = "bin/main.rs"
 test = false
 
-[[bin]]
-name = "rustc"
-path = "bin/rustc.rs"
-test = false
-
-[[bin]]
-name = "rustdoc"
-path = "bin/rustdoc.rs"
-test = false
-
 [[bin]]
```

Comment:
Maybe we should also do this for the other binaries? They get modified less often but I think they'll have the same issue.

Reply:
Done.